CHANGELOG.rst

```diff
@@ -31,6 +31,8 @@ Added
 
 - Tarantool Raft options are supported in argparse.
 
+- ``on_apply_config(conf, state)`` roles API.
```

Comment:
Let's stay it for internal usage. Nobody asked about such features. Let's stay it in "gray" zone and won't extend existing API